package/package.mk

```diff
@@ -2,7 +2,7 @@
 package-function: docker-build
 	yq e '.spec.image="${GHCR_IMG}"' package/crossplane.yaml.template > package/crossplane.yaml
 	rm -f package/*.xpkg
-	go run github.com/crossplane/crossplane/cmd/crank xpkg build -f package --verbose --embed-runtime-image=${GHCR_IMG} -o package/package-function-appcat.xpkg
+	go run github.com/crossplane/crossplane/cmd/crank@v1.16.0 xpkg build -f package --verbose --embed-runtime-image=${GHCR_IMG} -o package/package-function-appcat.xpkg
```

Comment:
If version is not specified wouldn't it get the latest version?

Reply:
It's a long story, but no. It would get the version defined in the modules `go.mod`. Usually this is fine, but we can't currently use this version of crank via `go.mod` because it has some dependency that conflicts with another dependency in `functions-go-sdk`.